supervisor/docker/interface.py

```diff
@@ -95,6 +95,17 @@ def _install(
 
         _LOGGER.info("Pull image %s tag %s.", image, tag)
         try:
+            # If the image name contains a path to a registry, try to find its credentials and log in
+            if "/" in image:
```

Comment:
Every image has an `/` in, but you can use the regex from validation now. Please offload the part above into an internal function `_docker_login` on the interface.

Reply:
So I guess it should be a regex that matches at least two `/` in a string, right? Should the condition also go to the `_docker_login` or just anything below it?

Reply:
The regex they we use for config validation can be use here to detect an host inside image url